Utilities/Tools/jobutils2.sh

```diff
@@ -395,10 +395,13 @@ getNumberOfPhysicalCPUCores() {
     fi
   else
     # Do something under GNU/Linux platform
-    CORESPERSOCKET=`lscpu | grep "Core(s) per socket" | awk '{print $4}'`
-    SOCKETS=`lscpu | grep "Socket(s)" | awk '{print $2}'`
+    #
+# Gets the cores per socket by counting unique cores on socket 0.
+# Gets sockets by counting unique socket ids. The grepping is done in any case  to avoid matching comments. 
```

Comment:
```suggestion
    # Gets the cores per socket by counting unique cores on socket 0.
    # Gets sockets by counting unique socket ids. The grepping is done in any case  to avoid matching comments. 
```